src/Shell/Task/DumpTask.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * CakePHP(tm) : Rapid Development Framework (http://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @link          http://cakephp.org CakePHP(tm) Project
+ * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Migrations\Shell\Task;
+
+/**
```

Comment:
`use igrations\Shell\Task\CommandTask` ?

Reply:
Its the same namespace, so no need to.

Reply:
Actually forbidden even with unused use sniffs :)

Reply:
it is needed for increase readability 
+
avoid of future problems like moving class to place 
I see this problem when working on moving cakephp's HTTP  class

Reply:
@dereuromark  what says cakephp-codesniffer?

Reply:
It doesnt have the advanced sniffs other sniff repos has :) But it is not needed and should be removed. IDEs would also mark it as unneeded.

Reply:
@saeideng I can't recall any class in CakePHP with use statemtents for the classes found in the namespace.